en/deploy/README.md

```diff
@@ -221,14 +221,14 @@ Now our code is on PythonAnywhere, our virtualenv is ready, and the database is
 
 Click back to the PythonAnywhere dashboard by clicking on its logo, and then click on the **Web** tab. Finally, hit **Add a new web app**.
 
-After confirming your domain name, choose **manual configuration** (N.B. – *not* the "Django" option) in the dialog. Next choose **Python 3.6**, and click Next to finish the wizard.
+After confirming your domain name, choose the **manual configuration** (N.B. – *not* the "Django" option) in the dialog. Next, choose **Python 3.6**, and click Next to finish the wizard.
```

Comment:
The additional "the" here, to me, seems very awkward. The comma is probably a good idea.

Reply:
Actually, this sentence doesn't seem to be in the file any more?